namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-preprod/resources/s3.tf

```diff
@@ -0,0 +1,33 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/master/example
+ */
+module "book_a_secure_move_documents_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=3.4"
+
+  team_name              = "${var.team_name}"
+  business-unit          = "Digital and Technology"
+  application            = "${var.application}"
+  infrastructure-support = "${var.infrastructure-support}"
+
+  is-production          = "${var.is-production}"
+  environment-name       = "${var.environment-name}"
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = "aws.london"
+  }
+}
+
+resource "kubernetes_secret" "book_a_secure_move_documents_s3_bucket" {
+  metadata {
+    name      = "book_a_secure_move_documents_s3_bucket"
```

Comment:
Hi @spikeheap , plan pipeline failed, it is complaining about the name, i cannot have "_" in a name as shown in the error.
`https://concourse.cloud-platform.service.justice.gov.uk/builds/37177`